### PR TITLE
Update report_infringement.md

### PR DESCRIPTION
**Pull request by mafeld**
Update for Mozilla.social form

### CODE REVIEW

File: en/report_infringement.md [hunk 1]
```diff
@@ -17,15 +17,15 @@ Your DMCA Notice must include all of the following information:
...
+To submit notices related to content on Mozilla.social, please use [this form](https://reports.mozilla.social/infringement-form). For other Mozilla products and sites, the quickest way is to email it to [dmcanotice@mozilla.com](mailto:dmcanotice@mozilla.com). If you prefer, you can send it to:
 
 Mozilla  
 Attn: Mozilla’s Designated DMCA Agent  
 149 New Montgomery St.  
 4th Floor  
 San Francisco, CA 94105  
 
-Mozilla’s Designated Agent’s fax is 650-903-0875 and phone is 650-903-0800.
+Mozilla’s Designated Agent’s phone is 650-903-0800.
```
Comment (peiying2) [inline en/report_infringement.md:28]: Do we need to add a `+1` for the phone number?

Comment (mafeld) [reply en/report_infringement.md]: done

**Review by peiying2 — APPROVED** [review]

**Comment by flodolo** [comment]
@peiying2 @mafeld 
What's the plan with this PR?

**Comment by mafeld** [comment]
I think we're going to wait and bundle this with some other updates on Nov 1.

File: en/report_infringement.md [hunk 2]
```diff
@@ -17,15 +17,15 @@ Your DMCA Notice must include all of the following information:
 
 ### Where do you email/send/fax the DMCA Notice?
```
Comment (awmcd) [inline en/report_infringement.md:18]: @mafeld I missed this in the doc version, but can we update the heading too?
```suggestion
### Where do you send the DMCA Notice?
```

**Pull request merged**